test/rendering.test.js

```diff
@@ -0,0 +1,351 @@
+import { beforeAll, describe, expect, it } from 'vitest';
+import { mount } from "@vue/test-utils";
+import App from '../src/components/App.vue';
+import store from '../src/store.js';
+import { handleMessage } from '../src/handleMessage.js';
+
+const USER_1 = {
+  username: 'user1',
+  badges: {
+    broadcaster: '1',
+  },
+};
+const USER_2 = {
+  username: 'user2',
+};
+const USER_3 = {
+  username: 'user3',
+};
+
+const INITIAL_POLL_STATE = {
+  active: false,
+  visible: false,
+  title: 'Poll',
+  options: {},
+  userVotes: {},
+};
+
+const pollWrapper = mount(App, {
+  attachTo: 'body' // 'attachTo' is necessary for isVisible() to work properly
+});
+
+describe('rendering', function () {
+
+  describe('when receiving a simple poll start command', function () {
+    beforeAll(function () {
+      Object.assign(store, structuredClone(INITIAL_POLL_STATE));
```

Comment:
Yayyyy! Erstes Vorkommen von `structuredClone` was ich in freier Wildbahn sehe 😍